ui/components/multichain/nft-item/nft-item.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import NftDefaultImage from '../../app/nft-default-image/nft-default-image';
+import {
+  AvatarNetwork,
+  BadgeWrapper,
+  BadgeWrapperAnchorElementShape,
+} from '../../component-library';
+import {
+  BackgroundColor,
+  DISPLAY,
+  Size,
+} from '../../../helpers/constants/design-system';
+import Box from '../../ui/box/box';
+
+export const NftItem = ({
+  alt,
+  backgroundColor,
+  name,
+  src,
+  networkName,
+  networkSrc,
+  tokenId,
+  onClick,
+}) => {
+  return (
+    <Box data-testid="nft-item">
+      <BadgeWrapper
+        badge={
+          <AvatarNetwork
+            size={Size.SM}
+            name={networkName}
+            src={networkSrc}
+            borderColor={BackgroundColor.backgroundDefault}
+            borderWidth={2}
+            data-testid="nft-network-badge"
+          />
+        }
+        anchorElementShape={BadgeWrapperAnchorElementShape.rectangular}
+        positionObj={{ top: -4, right: -4 }}
```

Comment:
We might need to change the values here since the badge position seems to be different from the design.

### Design
![Screenshot 2023-05-19 at 11 42 15 AM](https://github.com/MetaMask/metamask-extension/assets/39872794/f3dbabfc-b7cd-4a18-bdc0-e2ca534c19ca)

### Current UI
![Screenshot 2023-05-19 at 11 41 57 AM](https://github.com/MetaMask/metamask-extension/assets/39872794/5b0641e8-9a85-4ed2-9eff-1dcc0683656c)

Reply:
<img width="347" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/61c57c39-0d68-4b84-9c56-beee0cc8a3b1">

This is the design I'm seeing on the figma

Reply:
I was looking at the screenshot in ticket. But yeah, this looks good to me